nhrpd/netlink_arp.c

```diff
@@ -191,6 +191,11 @@ int nhrp_neighbor_operation(ZAPI_CALLBACK_ARGS)
 			       "Netlink: update binding for %pSU dev %s from c %pSU peer.vc.nbma %pSU to lladdr %pSU",
 			       &addr, ifp->name, &c->cur.remote_nbma_natoa,
 			       &c->cur.peer->vc->remote.nbma, &lladdr);
+
+			if (lladdr.sa.sa_family == AF_UNSPEC)
+			  /* nothing from zebra, so use nhrp peer */
+			  lladdr = c->cur.peer->vc->remote.nbma;
```

Comment:
please fix the indentation!

Reply:
sigh, not sure what happened there.  was right then did a something that broke it.  Fixed (again) now. My apologies.